modules/apps/client-extension/client-extension-type-impl/src/main/java/com/liferay/client/extension/type/internal/factory/CETFactoryImpl.java

```diff
@@ -264,19 +264,12 @@ protected void activate() {
 		).put(
 			ClientExtensionEntryConstants.TYPE_THEME_CSS,
 			new ThemeCSSCETImplFactoryImpl(_jsonFactory)
-		).put(
-			ClientExtensionEntryConstants.TYPE_THEME_SPRITEMAP,
-			new ThemeSpritemapCETImplFactoryImpl()
 		).put(
 			ClientExtensionEntryConstants.TYPE_THEME_FAVICON,
 			new ThemeFaviconCETImplFactoryImpl()
-
-		// TODO
-
-		/*).put(
```

Comment:
@thiago-buarqque I think because this was commented out, this actually isn't a breaking change. Could you ask verify this with some other devs? This feature wasn't available for use by customers, so I don't think we can be breaking anything for them.

If that ends up being the case can you break this up into multiple commits, at least the language files?

Reply:
@ethib137 
> I think because this was commented out, this actually isn't a breaking change. Could you verify this with some other devs? This feature wasn't available for use by customers, so I don't think we can be breaking anything for them.

That's true, but the ThemeJSCET entity was available throughout the client-extension-api, meaning if I remove it, it breaks the api and the source formatter will complain and we can't (shouldn't) do anything to ignore it.

> If that ends up being the case can you break this up into multiple commits, at least the language files?
I'm not sure 

I see some relevance in keeping the breaking change together in a single commit, since they're related to the same functionality , so searching for it on git history is easier. Still, should I really split into multiple commits?

Reply:
Great. Thanks!